README.adoc

```diff
@@ -15,25 +15,55 @@ ifndef::env-github[]
 image::images/Ui.png[width="600"]
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
-* What's different from https://github.com/se-edu/addressbook-level3[level 3]:
-** A more sophisticated GUI that includes a list  panel and an in-built Browser.
-** More test cases, including automated GUI testing.
-** Support for _Build Automation_ using Gradle and for _Continuous Integration_ using Travis CI.
+* This is a relationship tracker based on a desktop Address Book application. It has a GUI but most of the user interactions
+happen using a CLI (Command Line Interface).
+* It is primarily targeted at *private investigators* who wish to find a way to maintain their ever-growing and ever-changing
+information base easily.
+
+== Why Intelli?
+
+Wondering why you should use Intelli? +
+Here are three extra-special features Intelli offers that may convince you why:
+
+* Add named relationships between your contacts
+
+  Intelli enables you to easily keep track of who's who in vast and complicated social networks
```

Comment:
This indentation style shows up as a code block. Is this intended? Seems like a regular indented bullet point would work better.

Reply:
Haha actually I did look at the file after converting it to HTML and rendering it and I thought it looked okay, but then again that may be because I was the one who edited it so I'm blind to its oddity. I will update this in a bit